code/style.css

```diff
@@ -1,4 +1,191 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  margin: 0;
+  font-family: 'Oxygen', sans-serif;
+  background-color: #9ba4b4;
 }
+
+h1 {
+  font-family: 'Noto Sans JP', sans-serif;
+  font-size: 30px;
+  font-weight: 100;
+}
+
+h2 {
+  font-size: 20px;
+  font-weight: 100;
+}
+
+h3 {
+  font-size: 15px;
+  font-weight: 100;
+  margin: 5px 0;
+}
+
+p {
+  font-size: 15px;
+  font-family: 'Noto Sans JP', sans-serif;
+  margin: 0 0 25px 0;
+}
+
+/* HEADER STYLING */
+.header {
+  background: #14274e;
+  color: #f1f6f9;
+  display: flex;
+  align-items: center;
+}
+
+.logo-image {
+  height: 50px;
+  width: 50px;
+  margin: 0 10px 0 15px;
+}
+
+nav {
+  display: none;
+}
+
+nav a {
+  color: #f1f6f9;
+  text-decoration: none;
+  font-size: 15px;
+  padding: 0 15px;
+}
+
+nav a:hover {
+  background: #394867;
+}
+
+.menu-link {
+  margin-left: auto;
+  padding-right: 15px;
+}
+
+.menu-bars {
+  height: 25px;
+  width: 25px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.menu-bars div {
+  background-color: #f1f6f9;
+  width: 25px;
+  height: 2px;
+}
+
+/* BIG NEWS STYLING */
+.big-news {
+  display: flex;
+  flex-direction: column;
+  background: #394867;
+  color: #f1f6f9;
+}
+
+.big-news img {
+  max-width: 100%;
+  max-height: 100%;
+  display: block;
+}
+
+.big-news-text {
+  padding: 0 30px 5px;
+}
+
+/* SMALL NEWS STYLING */
+.small-news {
+  display: flex;
+  flex-direction: column;
+  background-color: #f1f6f9;
+  color: #14274e;
+}
+
+.small-news img {
+  width: 100%;
+}
+
+.small-news img:hover {
+  box-shadow: 0px 5px 0px 5px #394867;
+}
```

Comment:
This box-shadow kind of overlaps on the sibling images. Maybe you can allocate some space in between the elements or just have the box shadow at the bottom